hack/verify-mocks.sh

```diff
@@ -25,62 +25,26 @@ set -o nounset
 set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-export KUBE_ROOT
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
 # Explicitly opt into go modules, even though we're inside a GOPATH directory
 export GO111MODULE=on
 
-_tmp="${KUBE_ROOT}/_tmp"
-mkdir -p "${_tmp}"
+kube::util::ensure_clean_working_dir
 
-"${KUBE_ROOT}/hack/update-mocks.sh"
+_tmpdir="$(kube::realpath "$(mktemp -d -t "$(basename "$0").XXXXXX")")"
+git worktree add -f -q "${_tmpdir}" HEAD
```

Comment:
this has some interesting side-effects, because `_output` is not re-used across the worktrees, so we throway binaries in _output/go/bin and other caching.